### ISSUE TITLE
JSON schematic system

### ISSUE THREAD

**Issue by asiekierka** [BuildCraft/BuildCraft#3058]
TODO:

[ ] Add "/items/" for custom item handling
[ ] Make storeRequirements(...) use a FakeWorld with the NBT-transformed tile upon blueprint creation
[ ] Figure out best layout for JSONs
[ ] Fix hack detection dropping items

**Comment by Lunatrius** [comment]
Messed with this a few months back, never really finished it.

Old concept gists if they're of any use: [JSON](https://gist.github.com/Lunatrius/7588a99da23e2aeb1a7a) / [X PixMap](https://gist.github.com/Lunatrius/43fafb371b94c79062d0)

**Comment by Kubuxu** [comment]
@Lunatrius this is something different, it isn't about schematic saving formant but JSON API for declaring support with BC Builders.

See: BuildCraft/BuildcraftCompat#44

**Comment by Lunatrius** [comment]
Oh, I see, derp on my part...

**Comment by AEnterprise** [comment]
is this still planned?

**Comment by AlexIIL** [comment]
Yes, although I'm evaluating rewriting the schematic API itself to allow for different rendering effects and better support for multi-stage builds (like building a chest slowly, or allowing parts of a pipe to be placed even before other parts are available, and "pausing" the pipe so it doesn't tick until its ready).
